### ISSUE TITLE
error: cannot find symbol class ReadableArray

### ISSUE THREAD

**Issue by damusnet** [corymsmith/react-native-fabric#31]
Hello,

I had `react-native-fabric 0.1.0` working fine with `react-native 0.25.1`. Now I just tried updating to `react-native-fabric 0.2.0` and it gives me the following error. Do you know what is happening? Thanks for your help.

```
:react-native-fabric:compileReleaseJavaWithJavac - is not incremental (e.g. outputs have changed, no previous execution, etc.).
/Users/damien/Sites/digisquare-mobile/digisquare/node_modules/react-native-fabric/android/src/main/java/com/smixx/fabric/SMXCrashlytics.java:84: error: cannot find symbol
    public void recordCustomExceptionName(String name, String reason, ReadableArray frameArray) {
                                                                      ^
  symbol:   class ReadableArray
  location: class SMXCrashlytics
/Users/damien/Sites/digisquare-mobile/digisquare/node_modules/react-native-fabric/android/src/main/java/com/smixx/fabric/SMXCrashlytics.java:87: error: cannot find symbol
            ReadableMap map = frameArray.getMap(i);
            ^
  symbol:   class ReadableMap
  location: class SMXCrashlytics
2 errors
:react-native-fabric:compileReleaseJavaWithJavac FAILED

FAILURE: Build failed with an exception.

* What went wrong:
Execution failed for task ':react-native-fabric:compileReleaseJavaWithJavac'.
> Compilation failed; see the compiler error output for details.
```

**Comment by corymsmith** [comment]
I'm currently investigating

**Comment by corymsmith** [comment]
Just published 0.2.1 which should fix this, let me know

**Comment by damusnet** [comment]
Hi, thanks for looking into it so quickly. Now the project compiles ok, but my app crashes on startup, so I can't say for sure. I tried going back to `react-native-fabric 0.1.0` and it still crashes on startup, so it looks unrelated, although I didn't make any other change since it used to work (yesterday). Anyway, thanks again!

**Comment by corymsmith** [comment]
What's the crash?

**Comment by damusnet** [comment]
One error said something about `ERROR: APK path is not specified for module "app"`
Another was like `Google Play services out of date. Requires 7095000 but found 6774470`
Also `GoogleService failed to initialize, status: 10, Missing an expected resource: 'R.string.google_app_id' for initializing Google services`
It more likely seemed related to the `react-native-maps` package and its Google Maps configuration.
Like I said, it occurred also when rolling back to `react-native-fabric 0.1.0`.

In the end, I ended up adding a [Google Project Number](http://stackoverflow.com/a/34614236/233404) in my settings (presumably for Google Maps, although it used to work just fine), and more importantly, I adb uninstalled my app from my emulator before trying again. Not exactly sure what did it, but after that the app launched fine again. Thus I was able to upgrade `react-native-fabric` to 0.2.1, compile, install and launch just fine.

Thanks for your support.
